### DISCUSSION TITLE
1.08 X64的程序包我安装到WIN11 22H2企业版怎么安装不上？？

### DISCUSSION THREAD

**Discussion by myzhongqiang** [JasonWei512/EnergyStarX#53] [General]
应用安装失败，错误消息: Windows 无法安装程序包 14463NickJohn.214685D9CE7ED，因为此程序包需要体系结构 ARM64，但此计算机具有体系结构 x64。 (0x80073d10)

**Comment by JasonWei512** [comment]
你是从哪里下载的安装包？如果是果核剥壳之类的第三方下载站，那么是上传的人传错了，传了 arm64 的包。

推荐从微软商店安装，可以自动辨别你的电脑是 x64 还是 arm64，而且装了之后如果有新版本会通过商店自动静默更新的。商店搜”能源之星X“就行。

如果用不了微软商店，那么从 https://github.com/JasonWei512/EnergyStarX/releases 下载 x64 安装包。

**Comment by myzhongqiang** [comment]
谢谢 在商店更新下载又可以了

从 Windows 版邮件<https://go.microsoft.com/fwlink/?LinkId=550986>发送

________________________________
发件人: JasonWei512 ***@***.***>
发送时间: Monday, March 13, 2023 6:09:42 PM
收件人: JasonWei512/EnergyStarX ***@***.***>
抄送: myzhongqiang ***@***.***>; Author ***@***.***>
主题: Re: [JasonWei512/EnergyStarX] 1.08 X64的程序包我安装到WIN11 22H2企业版怎么安装不上？？ (Discussion #53)


你是从哪里下载的安装包？如果是果核剥壳之类的第三方下载站，那么是上传的人传错了，传了 arm64 的包。

推荐从微软商店安装，可以自动辨别你的电脑是 x64 还是 arm64，而且装了之后如果有新版本会通过商店自动静默更新的。商店搜”能源之星X“就行。

如果用不了微软商店，那么从 https://github.com/JasonWei512/EnergyStarX/releases 下载 x64 安装包。

―
